integreat_cms/cms/templates/pois/poi_form_sidebar/action_box.html

```diff
@@ -47,6 +47,24 @@
                         <i icon-name="pen-square" class="mr-2"></i> {{ event.best_translation.title }}
                     </a>
                 {% endfor %}
+            {% elif poi_form.instance.contacts.exists %}
+                <div class="bg-orange-100 border-l-4 border-orange-500 text-orange-700 px-4 py-3 mb-5"
+                     role="alert">
+                    <p>
+                        {% translate "You cannot archive a location which is referenced by a contact." %}
+                        <br />
+                        {% blocktranslate count counter=poi_form.instance.contacts.count trimmed %}
+                            To archive this location, you have to delete this contact first:
+                        {% plural %}
+                            To archive this location, you have to delete these contacts first:
+                        {% endblocktranslate %}
+                    </p>
+                </div>
+                {% for contact in poi_form.instance.contacts.all %}
+                    <a href="#" class="block pt-2 hover:underline">
```

Comment:
@MizukiTemma could you please set the real URL for editing a contact in your PR? :)

Reply:
Thank you for letting me know it 👍 I'll add it once this PR gets merged 😸